airbyte-webapp/src/components/connection/DestinationNamespaceModal/DestinationNamespaceModal.tsx

```diff
@@ -48,7 +48,7 @@ export const DestinationNamespaceModal: React.FC<DestinationNamespaceModalProps>
   return (
     <Formik
       initialValues={{
-        namespaceDefinition: initialValues?.namespaceDefinition ?? NamespaceDefinitionType.source,
+        namespaceDefinition: initialValues?.namespaceDefinition ?? NamespaceDefinitionType.destination,
```

Comment:
This will correctly set `NamespaceDefinitionType.destination` as the default, but it still appears as the second option in the list due to the order in the HTML below:

![image](https://user-images.githubusercontent.com/7550957/210755403-5cfec363-b6c7-428d-a050-621d89ed8c29.png)

To make it the first option, you could move the destination `<Field>` component (lines 83-98) before the source component (lines 67-82).

Note: this is only visible in the upcoming stream table redesign, so you need to set `REACT_APP_NEW_STREAMS_TABLE=true` in `airbyte-webapp/.env.development` before spinning up the server with `npm run start` or `npm run start:cloud` to see this new modal UI.